examples/index.json

```diff
@@ -578,7 +578,7 @@
                 "isLearnByExample": true
             },
             {
-                "name": "Querying with streams",
+                "name": "Query with streams",
```

Comment:
Here again I think "Querying" is more suitable.

Reply:
Reverted.